src/main/java/net/mcreator/ui/dialogs/JavaModelAnimationEditorDialog.java

```diff
@@ -111,10 +112,9 @@ public static String openAnimationEditorDialog(MCreator mcreator, String modelSo
 		if (animations.size() > 10)
 			sp.setPreferredSize(new Dimension(450, 400));
 
-		int opt = JOptionPane
-				.showOptionDialog(mcreator, sp, "Model animation definition wizard", JOptionPane.YES_NO_OPTION,
-						JOptionPane.PLAIN_MESSAGE, null,
-						new String[] { "Set new animations", "Keep current animations" }, "Set animations");
+		int opt = JOptionPane.showOptionDialog(mcreator, sp, L10N.t("dialog.animation_editor.title_wizard"),
+				JOptionPane.YES_NO_OPTION, JOptionPane.PLAIN_MESSAGE, null,
+				new String[] { "Set new animations", "Keep current animations" }, "Set animations");

```

Comment:
You can translate these too

Reply:
I don't think I'm allowed to translate string lists

Reply:
These are strings for the option pane buttons and are converted to int opt so you should be fine.

Reply:
Oh okay

Reply:
Done!